internal/api/integrations/k8s/k8s.go

```diff
@@ -76,16 +85,44 @@ func (k *K8sClient) ListAll(namespace string) ([]models.Release, error) {
 				Sumologic: annotations[annotationFor("sumologic")],
 			},
 			Code: models.SourceCode{
-				Github: annotations[annotationFor("code")],
+				Github: findVCSRepo(codeURL),
+				Ref:    image.Tag,
 			},
 			Artifacts: models.Artifacts{
 				Chart: models.HelmArtifact{
 					Path:    r.Spec.Chart.Path,
 					Version: r.Spec.Chart.Revision,
 				},
+				Docker: models.DockerArtifact{
+					Image: image.Repository,
+					Tag:   image.Tag,
+				},
 			},
 		})
 	}
 
 	return releases, nil
 }
+
+func findVCSRepo(url string) string {
+	arr := strings.Split(url, "/")
```

Comment:
I think we should implement this func by [parsing](https://golang.org/pkg/net/url/#Parse) the URL (to validate it's a real URL). Then getting the repo split from the `URL.Path`